Library/Formula/dmd.rb

```diff
@@ -65,6 +65,31 @@ def install
     end
   end
 
+  # Previous versions of this formula may have left in place an incorrect
+  # dmd.conf.  If it differs from the newly generated one, move it out of place
+  # and warn the user.
+  # This must be idempotent because it may run from both install() and
+  # post_install() if the user is running `brew install --build-from-source`.
+  def install_new_dmd_conf
+    conf = etc/"dmd.conf"
+    # Iff the new file differs from conf, etc.install drops it here:
```

Comment:
I meant [iff], but I can change it if you like.

[iff]: http://en.wikipedia.org/wiki/If_and_only_if